app/workers/document_list_export_worker.rb

```diff
@@ -17,7 +17,10 @@ def send_mail(csv, user, filter)
   end
 
   def create_filter(filter_options, user)
-    Admin::EditionFilter.new(Edition, user, filter_options.symbolize_keys.merge(include_unpublishing: true))
+    Admin::EditionFilter.new(Edition, user, filter_options.symbolize_keys.merge(
+                                              include_unpublishing: true,
+                                              include_locked_documents: false,
+    ))
```

Comment:
The formatting of this one looks odd to me - might be neater with each argument on its own line?

Reply:
Yeah the linter complained a lot about this, never would have put stuff here on my own. Will mutliline this now :)